### ISSUE TITLE
on the tad features extraction, is image normalization required?

### ISSUE THREAD

**Issue by auzxb** [OpenGVLab/VideoMAEv2#62]
on the train stage，the images have to be loaded in to a range of [0, 1] and then normalized using mean = [0.485, 0.456, 0.406] and std = [0.229, 0.224, 0.225]，is image normalization required when extracting tad features？

**Comment by hywang66** [comment]
I have the same question here. Can anyone answer it?

**Comment by oooolga** [comment]
Same question 👀

**Comment by songweige** [comment]
I have the same question here. Seems that there is a discrepancy in the preprocessing code between the [training](https://github.com/OpenGVLab/VideoMAEv2/blob/bddefef66ea189a7d83f56bf1769b25594bedfdc/dataset/pretrain_datasets.py#L36) and [inference](https://github.com/OpenGVLab/VideoMAEv2/blob/bddefef66ea189a7d83f56bf1769b25594bedfdc/extract_tad_feature.py#L105).
